src/backend/aspen/auth/auth0_management.py

```diff
@@ -143,6 +143,18 @@ def add_org_member(self, org: Auth0Org, user_id: str) -> None:
             org["id"], user_id, {"roles": [member_role["id"]]}
         )
 
+    def add_org_roles(self, org: Auth0Org, user_id: str, roles: List[str]) -> None:
+        roleids = [self.get_auth0_role(role)["id"] for role in roles]
```

Comment:
this is the nittiest nit, but can this variable be named `role_ids`? for some reason `roleids` is upsetting to me